parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -165,7 +175,7 @@ pub mod pallet {
 
 	#[pallet::storage]
 	pub(super) type FinalizedBeaconHeaderSlots<T: Config> =
-		StorageValue<_, BoundedVec<u64, T::MaxFinalizedHeaderSlotArray>, ValueQuery>;
+		StorageValue<_, BoundedVec<(u64, H256), T::MaxFinalizedHeaderSlotArray>, ValueQuery>;
```

Comment:
Just reminder, require associated change in relayer side [here](https://github.com/Snowfork/snowbridge/blob/4f3b5910edca114059ace9de130e1524d0a20625/relayer/chain/parachain/writer.go#L228-L246)